index.html

```diff
@@ -0,0 +1,200 @@
+              <h3>Exhibition</h3>
+            </div>
+            <p>
+              40+ Pharmaceutical & FMCG companies have confirmed their
+              participation as exhibitors at the conference.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Excursion</h3>
+            </div>
+            <p>
+              Explore the vast Historic collections at the national Museum, Walk
+              through a range of art works at the Nike Art Garlary
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Workshop</h3>
+            </div>
+            <p>
+              The mandatory continuing professional development program that
+              will be hosted in this year’s conference is focused on Vaccination
+              Training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Novelty Match</h3>
+            </div>
+            <p>
+              There will be a Novelty Match between the different state arm of
+              the association.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-palette"></i>
+              <h3>Group Dynamics</h3>
+            </div>
+            <p>
+              Every state arm of the association will be required to exhit or
+              showcase their progress and activites within the past one year
+            </p>
+          </div>
+          <div class="join">
+            <a href="#">Join ACPN 2023.</a>
+          </div>
+        </div>
+        <p class="show">SEE THE WHOLE PROGRAM</p>
+      </div>
+    </section>
+
+    <!------------------------------PROGRAMS  END--------------------------------->
+
+    <!------------------------------SPEAKERS  STARTS------------------------------>
+    <section class="speakers">
+      <div class="container">
+        <h2>Featured Speakers</h2>
+        <hr />
+        <div class="guest-speakers"></div>
+      </div>
+    </section>
+    <!------------------------------SPEAKERS  END--------------------------------->
+
+    <!------------------------------PARTNER  START-------------------------------->
+    <section class="partner sponsor" id="sponsor">
+      <div class="container">
+        <h4>Partners</h4>
+        <hr />
+        <div class="logos logo-home">
+          <p class="sponsor1">daumkakao</p>
+          <img src="images/coke.png" alt="" />
+          <img src="images/facebook.png" alt="" />
+          <img src="images/amazon.png" alt="" />
+          <img src="images/mozilla.png" alt="" />
+          <img src="images/google.svg" alt="" />
+        </div>
+      </div>
```

Comment:
- [ ] The previous reviewer asked that you use the best practices to semantically fix this layer user `ul` and `li` tags respectively. This is a suggestion to help you follow best practices, I'll also advise that you use a `span` for the `sponsor1` class instead of a `p` tag as it's not a paragraph. 

## Sample solution:
```
<div class="logos logo-home">
  <ul class="sponsor-list">
    <li class="sponsor-item"><span class="sponsor1">daumkakao</span></li>
    <li class="sponsor-item"><img src="images/coke.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/facebook.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/amazon.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/mozilla.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/google.svg" alt="" /></li>
  </ul>
</div>
```